### DISCUSSION TITLE
I urgently need ISO for version 2.3.120

### DISCUSSION THREAD

**Discussion by tiger-greenhell** [Security-Onion-Solutions/securityonion#11401] [General]
HI all.

I urgently need ISO version 2.3.120 due to reinstallation of one node 2.3.120 grid.

ISO download link on page https://github.com/Security-Onion-Solutions/securityonion/blob/2.3.120/VERIFY_ISO.md does not exist

result:

code | "not_found"
message | "File with such name does not exist."
status | 404

Thanks

**Comment by dougburks** [comment]
Security Onion 2.3.120 is over a year old:
https://blog.securityonion.net/2022/04/security-onion-23120-now-available.html

We don't keep old ISOs around that long since folks should be upgrading to more recent versions. 

Have you considered updating to 2.3.260?
https://blog.securityonion.net/2023/06/security-onion-23260-now-available.html

**Reply by petiepooo** [reply]
Please remember that 2.3.130 is needed to upgrade from prior releases to releases after that.  I hope you at least keep that version around longer than a year so that people like the OP can still upgrade to the most current version.  Leaving just 2.3.130 and the latest 2.3 release gives stragglers like the OP a path forward without reinstalling.

**Reply by tiger-greenhell** [reply]
Fortunately, it is still possible to install a specific version via git

for example:

git clone --depth 1 --branch '2.3.120' https://github.com/Security-Onion-Solution/securityonion

this helped me to install another node of this version in the LAB environment

**Reply by Acewiza** [reply]
M$ owns/runs Git now, right?  That'll be the day when year-old software reaches the "urgent" stage with me.
